model/trace/database.yaml

```diff
@@ -561,46 +224,29 @@ groups:
              Format Reg-{D (Disabled discovery)}-S(application region)|L(List of preferred regions)|N(None, user did not configure it).
              Default value is "NS".
         examples: ['cosmos-netstandard-sdk/3.23.0\|3.23.1\|1\|X64\|Linux 5.4.0-1098-azure 104 18\|.NET Core 3.1.32\|S\|']
-      - id: connection_mode
-        type:
-          allow_custom_values: false
-          members:
-            - id: gateway
-              value: 'gateway'
-              brief: Gateway (HTTP) connections mode
-            - id: direct
-              value: 'direct'
-              brief: Direct connection.
-        brief: Cosmos client connection mode.
+        tag: call-level-tech-specific
+      - ref: db.cosmosdb.connection_mode
         requirement_level:
           conditionally_required: if not `direct` (or pick gw as default)
-      - id: container
-        type: string
-        brief: Cosmos DB container name.
+        tag: call-level-tech-specific
+      - ref: db.cosmosdb.container
         requirement_level:
           conditionally_required: if available
-        examples: 'anystring'
-      - id: request_content_length
-        type: int
-        brief: Request payload size in bytes
-      - id: status_code
-        type: int
-        brief: Cosmos DB status code.
-        examples: [200, 201]
+        tag: call-level-tech-specific
+      - ref: db.cosmosdb.request_content_length
+        tag: call-level-tech-specific
+      - ref: db.cosmosdb.status_code
         requirement_level:
           conditionally_required: if response was received
-      - id: sub_status_code
-        type: int
-        brief: Cosmos DB sub status code.
-        examples: [1000, 1002]
+        tag: call-level-tech-specific
+      - ref: db.cosmosdb.sub_status_code
         requirement_level:
           conditionally_required: when response was received and contained sub-code.
-      - id: request_charge
-        type: double
-        brief: RU consumed for that operation
-        examples: [46.18, 1.0]
+        tag: call-level-tech-specific
+      - ref: db.cosmosdb.request_charge
         requirement_level:
           conditionally_required: when available
+        tag: call-level-tech-specific
```

Comment:
what this tag means exactly?

Reply:
It's being used to filter attributes when rendering. Here, in particular, it allows to render attributes in the registry by technology groups.